.credo.exs

```diff
@@ -54,7 +54,7 @@
         {Credo.Check.Warning.UnusedTupleOperation},
         {Credo.Check.Warning.OperationWithConstantResult},
         {CredoEnvvar.Check.Warning.EnvironmentVariablesAtCompileTime},
-        {CredoFilenameConsistency.Check.Consistency.FilenameConsistency, excluded_paths: ["test/support", "priv", "rel", "mix.exs"]}
+        {CredoFilenameConsistency.Check.Consistency.FilenameConsistency, excluded_paths: ["mix.exs", "config/releases.exs", "priv", "rel", "test/support"]}
```

Comment:
I think we’ll be able to remove `"rel"` from the list.

Reply:
I also added the whole `config` folder, not only `config/releases.exs` from the exclusions since modules created in config script files are utilities, not "runtime" critical modules. Thoughts?